sampita/welcome-to-nashville
# made JS code modular

# Description

I made our code modular by changing our <script>main.js</> to type="module" and import/exporting all related .js files

# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-additional-styling`
4. Run http server and see if app is still fully functioning, check code to see if modular
